src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -329,8 +357,10 @@ void NotationViewInputController::doZoomToTwoPages()
 
     qreal viewWidth = m_view->width();
     qreal viewHeight = m_view->height();
-    qreal pageWidth = notationStyle()->styleValue(mu::engraving::Sid::pageWidth).toDouble() * mu::engraving::DPI;
-    qreal pageHeight = notationStyle()->styleValue(mu::engraving::Sid::pageHeight).toDouble() * mu::engraving::DPI;
+    qreal pageWidth = notationStyle()->styleValue(mu::engraving::Sid::pageWidth).toDouble() * mu::engraving::DPI + 2
+                      * MScore::horizontalPageGapOdd;
+    qreal pageHeight = notationStyle()->styleValue(mu::engraving::Sid::pageHeight).toDouble() * mu::engraving::DPI + 2
+                       * MScore::horizontalPageGapOdd;
```

Comment:
(Uncrustify broke these lines at the worst place possible...)
```suggestion
    qreal pageWidth = notationStyle()->styleValue(mu::engraving::Sid::pageWidth).toDouble() * mu::engraving::DPI 
                      + 2 * MScore::horizontalPageGapOdd;
    qreal pageHeight = notationStyle()->styleValue(mu::engraving::Sid::pageHeight).toDouble() * mu::engraving::DPI
                       + 2 * MScore::horizontalPageGapOdd;
```